pkg/security/util.go

```diff
@@ -2,6 +2,7 @@ package security
 
 const (
 	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
+	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
```

Comment:
That was good magic.  Not bad magic.

On Wed, Oct 7, 2015 at 3:20 PM, Clayton Coleman <ccoleman@redhat.com> wrote:

> Sounds magic
>
> On Oct 7, 2015, at 2:54 PM, Paul Weil <notifications@github.com> wrote:
>
> In pkg/security/util.go
> <https://github.com/openshift/origin/pull/4996#discussion_r41430503>:
>
> > @@ -2,6 +2,7 @@ package security
> >
> >  const (
> >  	UIDRangeAnnotation     = "openshift.io/sa.scc.uid-range"
> > +	FSGroupAnnotation      = "openshift.io/sa.scc.fsgroup"
>
> Hmm, ok. So how about the annotation is rebranded as supplemental groups
> and takes a string of comma delimited groups. Groups in the format of n-n
> can be parsed into an allowable range by policy strategies. When assigning
> an fs group we take the first gid or the first gid of the range. The
> allocator will still default this annotation to the first uid of the uid
> range but an admin can customize it.
>
> openshift.io/sa.scc.supplemental-groups = "1,2,3-10" - strategy
> effectively allows supplemental groups of 1-10 and uses 1 as fsgroup
>
> openshift.io/sa.scc.supplemental-groups = "3-10" - strategy allows
> supplemental groups of 3-10 and uses 3 as fsgroup
>
> openshift.io/sa.scc.supplemental-groups = "3-10,12-15" - strategy allows
> supplemental groups of 3-10 and 12-15 and uses 3 as fsgroup
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/openshift/origin/pull/4996/files#r41430503>.
>
>